WordPress/gutenberg
# Site Editor: Fix Classic Editor Block

## Description
Classic editor is right now broken in `master` due to missing `tinymce` scripts.
![image](https://user-images.githubusercontent.com/2256104/98905221-56e0f600-24bb-11eb-8f13-ad85c1546ed5.png)

This is easily fixed by enqueuing the `tinymce` scripts included with WordPress.

## How has this been tested?
- Open site editor
- Add Classic block
- Make sure it works

## Screenshots
**master**
![image](https://user-images.githubusercontent.com/2256104/98905399-a6272680-24bb-11eb-83c8-de5fc57bba79.png)


**this branch**
![image](https://user-images.githubusercontent.com/2256104/98905342-8ee83900-24bb-11eb-9091-745ad1d87528.png)


## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
